KubeArmor/core/hookHandler.go

```diff
@@ -0,0 +1,94 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2022 Authors of KubeArmor
+
+package core
+
+import (
+	"encoding/json"
+	"io"
+	"os"
+
+	"github.com/fsnotify/fsnotify"
+	tp "github.com/kubearmor/KubeArmor/KubeArmor/types"
+)
+
+func (dm *KubeArmorDaemon) HandleFile(file string) {
+	f, err := os.Open(file)
+	if err != nil {
+		dm.Logger.Errf("Failed to open file '%s': %v. Ensure the file exists and has appropriate permissions.", file, err.Error())
+	}
```

Comment:
What happens if the file is created later? if KubeArmor runs as a service in host os then the file would always be created later. We need to make sure that we wait for 5 mins before giving up on this. WDYT?